aries-site/src/examples/templates/feedback/ModalTaskFlowExample/ModalTaskFlowExample.js

```diff
@@ -0,0 +1,76 @@
+import { useContext, useEffect, useState } from 'react';
+import { AnnounceContext, Box, Button } from 'grommet';
+import { ModalDialog } from '../../ModalDialog';
+import { config, PowerOnFlow } from './PowerOnFlow';
+import { TaskFlowFeedback } from './TaskFlowFeedback';
+
+export const ModalTaskFlowExample = ({ containerRef }) => {
+  // containerRef is for demonstration purposes on this site. Most
+  // implementations should likely remove.
+  const [showModal, setShowModal] = useState(true);
+  const [showFeedback, setShowFeedback] = useState(false);
+  // announce when the layer opens
+  const announce = useContext(AnnounceContext);
+  const defaultTitle = config.title;
+  const defaultContent = (
+    <PowerOnFlow
+      onCancel={() => {
+        announce(`Power on device modal cancelled and closed.`, 'assertive');
+        setShowModal(false);
+      }}
+      onClose={() => {
+        setShowFeedback(true);
+      }}
+    />
+  );
+  const [title, setTitle] = useState(defaultTitle);
+  const [contents, setContents] = useState(defaultContent);
+
+  // set modal contents
+  useEffect(() => {
```

Comment:
do we need this useEffect or can we just have the setTitle and setContents in the `onClose` of the PowerOnFlow?

Reply:
from what I saw I would think we need this `useEffect` so that the `title` and `content` is reset based on the `showfeedback` state

Reply:
I think it is a matter of preference. I took it out and moved the set state calls within each onClose. See if you like that any better.